### PR TITLE
Added sensitivegroup command allowing to group scoring by name prefix.

### PR DESCRIPTION
**Pull request by agheata**
This introduces the command: `/example17/detector/sensitivegroup prefix`  (and the same for example14) allowing to sum-up hits for all placed volumes having the name starting with `prefix`.

Added also 2 new macros in Example17: example17_validation.in and lhcb_validation.in to be used for physics validation

### CODE REVIEW

**Comment by agheata** [comment]
For validation with more statistics, one can use directly `example17_validation.in` and `lhcb_validation.in` and change `/gun/number` to a larger number of tracks. There are 10 events by default on one thread and hits are printed, so based on the event-by-event scoring we can compute error bars running each macro just once per configuration. As configuration, we should at least have field on/off (the default is now off) and /gun/energy 10/100 GeV (default now is 10).

File: examples/Example17/AdeptIntegration.cu [hunk 1]
```diff
@@ -234,6 +249,7 @@ void AdeptIntegration::InitializeGPU()
     gpuState.allmgr_d.trackmgr[i]  = gpuState.allmgr_h.trackmgr[i]->ConstructOnDevice();
     gpuState.particles[i].trackmgr = gpuState.allmgr_d.trackmgr[i];
     COPCORE_CUDA_CHECK(cudaMalloc(&gpuState.allmgr_d.leakedTracks[i], QueueSize));
+    InitLeakedQueues<<<1, 1>>>(gpuState.allmgr_d, kCapacity);
```
Comment (hahnjo) [inline examples/Example17/AdeptIntegration.cu:252]: If you want to call this here, the kernel must only construct the current leaked queue, not all three of them.

**Review by hahnjo — COMMENTED** [review]
Now that I think about this, I'm not so sure anymore about using queues for leaked particles in `Example17`: How do you prevent the compaction from re-using the slots? (In `Example14` this could never happen because slots weren't re-used, so just storing indices was fine)

**Comment by agheata** [comment]
> Now that I think about this, I'm not so sure anymore about using queues for leaked particles in `Example17`: How do you prevent the compaction from re-using the slots? (In `Example14` this could never happen because slots weren't re-used, so just storing indices was fine)

Right, compaction should only happen after copying away the leaked tracks somewhere else. Which becomes more complicated...

**Comment by agheata** [comment]
@hahnjo I removed the "fix" part which is invalid due to the way leaked tracks may be overridden by compaction, the crash may also be related to this.

File: examples/Example17/AdeptIntegration.cu (hunk 1 above)
Comment (agheata) [reply examples/Example17/AdeptIntegration.cu]: removed from this PR

**Review by hahnjo — APPROVED** [review]

**Comment by hahnjo** [comment]
Oh, but CI failed...

**Comment by agheata** [comment]
should be fixed now, I will squash

**Pull request merged**